### DISCUSSION TITLE
Feedin Entity unavailable ?

### DISCUSSION THREAD

**Discussion by Captain0Kangaroo** [macxq/foxess-ha#47] [General]
I've got any error in my energy dash setup and was wondering what might be causing it? 
Running the latest beat version (v0.15-beta2)
<img width="359" alt="Screenshot 2022-02-10 at 13 10 15" src="https://user-images.githubusercontent.com/96144685/153415347-6a5a4c6c-7de7-4083-b391-5c4da21f0689.png">

**Comment by williamjeccles** [comment]
Had the same issue today, has returned but there have been blips in the data today.
